### ISSUE TITLE
Cannot build synthea on Java 13

### ISSUE THREAD

**Issue by bugspencor** [synthetichealth/synthea#648]
My Java version is 13 on Windows 64:

```
$ java -version
openjdk version "13.0.1" 2019-10-15
OpenJDK Runtime Environment AdoptOpenJDK (build 13.0.1+9)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 13.0.1+9, mixed mode, sharing)
```

Running "./gradlew build check test" gives me an error:

```
$ ./gradlew build check test
Starting a Gradle Daemon, 1 incompatible and 1 stopped Daemons could not be reused, use --status for details

FAILURE: Build failed with an exception.

* Where:
Settings file 'Y:\devel\3rd-party\synthea\settings.gradle'

* What went wrong:
Could not compile settings file 'Y:\devel\3rd-party\synthea\settings.gradle'.
> startup failed:
  General error during semantic analysis: Unsupported class file major version 57

  java.lang.IllegalArgumentException: Unsupported class file major version 57
        at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:184)
        at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:166)
        at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:152)
        at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:273)
...
```

Setting in gradle\wrapper\gradle-wrapper.properties:
`distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip`

solves that error, but leads to an error when building javadoc:

```$ ./gradlew javadoc

> Task :javadoc
Y:\devel\3rd-party\synthea\src\main\java\org\mitre\synthea\export\CDWExporter.java:41: warning: empty <p> tag
 * <p></p>
      ^
Y:\devel\3rd-party\synthea\src\main\java\org\mitre\synthea\helpers\InnerClassTypeAdapterFactory.java:100: error: heading used out of sequence: <H3>, compared to implicit preceding heading: <H1>
 * <h3>Registering Types</h3>
   ^
Y:\devel\3rd-party\synthea\src\main\java\org\mitre\synthea\modules\BloodPressureValueGenerator.java:11: warning: empty <p> tag
 * <p></p>
      ^
... (many more warnings)
           ^
Y:\devel\3rd-party\synthea\src\main\java\org\mitre\synthea\world\concepts\HealthRecord.java:161: warning: no @param for value
    public Observation(long time, String type, Object value) {
           ^
1 error
100 warnings

> Task :javadoc FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':javadoc'.
> Javadoc generation failed. Generated Javadoc options file (useful for troubleshooting): 'Y:\devel\3rd-party\synthea\build\tmp\javadoc\javadoc.options'

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.1.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 10s
4 actionable tasks: 2 executed, 2 up-to-date```

**Comment by djvreeman** [comment]
I'm having the same issue on my Mac. Tried a few different Gradle distribution versions, and either get SSL errors or the javadoc error that @bugspencor descibes above.

**Comment by jawalonoski** [comment]
@bugspencor thank you for the bug report and @djvreeman thank you for the confirmation.

I don't have a lot of cycles right now to chase this down, but we are very willing to take a pull request that updates grade to be compatible with Java 13 (that is apparently the source of the error).

**Comment by jawalonoski** [comment]
Fixed by #654
